raster/src/main/scala/geotrellis/raster/io/geotiff/writer/TiffTagFieldValue.scala

```diff
@@ -6,8 +6,10 @@ import geotrellis.raster.io.geotiff.tags.codes._
 import geotrellis.raster.io.geotiff.tags.codes.TagCodes._
 import geotrellis.raster.io.geotiff.tags.codes.TiffFieldType._
 import geotrellis.vector.Extent
-
 import java.nio.ByteOrder
+
+import geotrellis.raster.render.IndexedColorMap
```

Comment:
Are you using IntelliJ? Curious, a lot of people do, but most of the Azavea team doesn't...useful to know if we try it out and have questions :)